api/pkg/apis/v1alpha1/providers/target/k8s/k8s_test.go

```diff
@@ -154,7 +154,7 @@ func TestDeploymentToComponents(t *testing.T) {
 						},
 						{
 							Name:            "rocket",
-							Image:           "evaamscontreg.azurecr.io/rocket:detection",
+							Image:           "prom/prometheus",
 							ImagePullPolicy: "Always",
 							Args:            []string{"pipeline=3", "line=https://aka.ms/lineeast960", "cat=car person"},
```

Comment:
prom/prometheus doesn't support cmdline args like pipeline=xxxx line=xxxx, let's find two different images and better find one which needs command-line arguments.